src/styles/_catalog.scss

```diff
@@ -1,22 +1,22 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (max-width: 1024px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+  @media (max-width: 768px) {
+    grid-template-columns: repeat(2, 200px);
+  }
```

Comment:
check your break points:) 4 cards appear from 1025 px ,not 1024px. same thing with 768px.
<img width="859" alt="Screenshot 2021-10-21 at 21 27 23" src="https://user-images.githubusercontent.com/72105322/138336179-308f725e-0294-4ec5-88f1-cbbece81bbf1.png">

Reply:
hm, you are right!